cmd/oras/root/push.go

```diff
@@ -242,33 +242,41 @@ func doPush(dst oras.Target, pack packFunc, copy copyFunc) (ocispec.Descriptor,
 func updateDisplayOption(opts *oras.CopyGraphOptions, fetcher content.Fetcher, verbose bool, tracked track.GraphTarget) {
 	committed := &sync.Map{}
 
+	const (
+		Skipped   = "Skipped  "
+		Uploaded  = "Uploaded "
+		Exists    = "Exists   "
+		Uploading = "Uploading"
+		Uploaded2 = "Uploaded "
```

Comment:
What's the difference between `Uploaded` and `Uploaded2`? 🧐

Reply:
`Uploaded2` is never used and thus can be removed.

Reply:
I'm curious if this `const` block is somehow auto-generated or not?